cinema/tests/test_movie_viewset.py

```diff
@@ -0,0 +1,126 @@
+from django.test import TestCase
+from django.urls import reverse
+from rest_framework import status
+from rest_framework_simplejwt.tokens import RefreshToken
+from cinema.models import Movie, Genre, Actor
+from django.contrib.auth import get_user_model
+
+
+class MovieViewSetTestCase(TestCase):
+    def setUp(self):
+        self.genre1 = Genre.objects.create(name="Action")

```

Comment:
you can create fixtures.json file  in you cinema app with test data and then plug it in here like so : 
```
class MovieViewSetTestCase(TestCase):
    fixtures = ["cinema_fixture.json"]

It allows you to reuse test data in different tests + you will not have to create records in db explicitly